collector/settings.mdx

```diff
@@ -59,11 +59,13 @@ Common settings for configuring collector behavior, independent of the platform.
       </td>
     </tr>
     <tr>
-      <td>api_base_url (<code>PGA_API_BASEURL</code>)</td>
+      <td>api_base_url (<code>PGA_API_BASE_URL</code>)</td>
       <td>https://api.pganalyze.com</td>
       <td>
         Base URL for contacting the pganalyze API. You typically do not need to change this, unless you are
         running <a href="/docs/enterprise">pganalyze Enterprise Server</a>.
+        <br /><br />
+        Note: the env var was <code>PGA_API_BASEURL</code> prior to collector release v0.55.0.
```

Comment:
Maybe we can go slightly more verbose at the start of the sentence:

```suggestion
        Note: The name of the environment variable was <code>PGA_API_BASEURL</code> prior to collector release v0.55.0.
```

<img width="786" alt="Screenshot 2024-03-20 at 1 49 59 PM" src="https://github.com/pganalyze/pganalyze-docs/assets/7227/7c9fa098-2781-42e3-9170-d3a1ffd7f142">